src/codecs.ts

```diff
@@ -843,5 +843,32 @@ export const DataFlowCsvInput = t.intersection([
   t.record(t.string, t.string),
 ]);
 
+export const CookieCsvInput = t.intersection([
+  t.type({
+    /** The value of the cookie */
+    Name: t.string,
+    /** The CSV of purposes mapped to that cookie */
+    Purpose: t.string,
+  }),
+  t.partial({
+    /** The service that the cookie relates to */
+    Service: t.string,
+    /** Notes and descriptions for the cookie */
+    Notes: t.string,
+    /** Set of cookie owners */
+    Owners: t.string,
+    /** Set of cookie team owners */
+    Teams: t.string,
+    /** LIVE vs NEEDS_REVIEW aka Approved vs Triage  */
+    Status: valuesOf(ConsentTrackerStatus),
+  }),
+  // Custom attributes
+  t.record(t.string, t.string),
+]);
+
 /** Type override */
```

Comment:
nit: lets move this `export type DataFlowCsvInput = t.TypeOf<typeof DataFlowCsvInput>;` line to live alongside the `DataFlowCsvInput` variable